tabular/src/autogluon/tabular/models/_utils/rapids_utils.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict
+
+from autogluon.core.utils import get_gpu_count_torch
+
+
+class RapidsModelMixin:
```

Comment:
Is this model supposed to inherit `AbstractModel`? Seen couple super() calls, but this model is not inheriting any class.

Reply:
I don't believe Mixin's are supposed to inherit. @gradientsky thoughts?

Reply:
Yes, they not supposed to be inherit `AbstractModel`, it's an additional functionality mixed-into the main class via multiple inheritance. One thing: I see some versions implement `ABC`: https://docs.python.org/3/library/abc.html#abc.ABCMeta. Not sure if that's really necessary though.

Reply:
I don't think ABC is necessary, I tested with a dummy model using inheritance and it worked fine (tbh I'm unsure what ABC does and when it is necessary).